jetcd-core/src/test/resources/ssl/.gitignore

```diff
@@ -0,0 +1 @@
+cfssl/
```

Comment:
not needed?

Reply:
It is needed only to generate the certs and it is created by the script

Reply:
I see.